Dynatrace/dynatrace-operator
# Add OwnerReferences to the "dynatrace-webhook-certs" secret

# Description

`OwnerReferences` field of the `dynatrace-webhook-certs` secret  is set to `dynatrace-webhook` Deployment to allow k8s  automatically remove the secret when the operator is uninstalled.

Additionally `OwnerReferences` of the `dynatrace-activegate-internal-proxy` secret is set to `<Dynakube>`.

## How can this be tested?
Deploy operator on a cluster.
Install `<dynakube>` with ActiveGate enabled and proxy.
Check `OwnerReferences` of `dynatrace-activegate-internal-proxy` and `dynatrace-webhook-certs` secrets.
Delete the `<dynakube>` and uninstall the operator.
Both secrets should be automatically removed.

## Checklist
